application.py

```diff
@@ -661,20 +743,23 @@ def createproblem():
 
     # Modify problems table
     db.execute("INSERT INTO problems (id, name, description, hints, point_value, category, flag) VALUES (:id, :name, :description, :hints, :point_value, :category, :flag)",
-            id=problem_id, name=name, description=description, hints=hints, point_value=point_value, category=category, flag=flag)
-    db.execute("ALTER TABLE problems_master ADD COLUMN :problem_id boolean NOT NULL DEFAULT (0)", problem_id=problem_id)
+               id=problem_id, name=name, description=description, hints=hints, point_value=point_value, category=category, flag=flag)
+    db.execute("ALTER TABLE problems_master ADD COLUMN :problem_id boolean NOT NULL DEFAULT (0)",
+               problem_id=problem_id)
 
     # Go to problems page on success
     return redirect("/problems")
 
+
 @app.route("/admin/ban")
 @admin_required
 def ban():
     user_id = request.args.get("user_id")
     if not user_id:
         return "Must provide user ID"
 
-    banned_status = db.execute("SELECT banned FROM users WHERE id=:id", id=user_id)
+    banned_status = db.execute(
+        "SELECT banned FROM users WHERE id=:id", id=user_id)
```

Comment:
i'm a bit confused here,

on one line you split the query into two lines, first one contains the string, second one contains the parameter

then on the second instance, you just moved both the query and parameter to a second line

any reason for this?